draft-ietf-webpush-protocol.xml

```diff
@@ -1137,6 +1147,12 @@ HEADERS      [stream 82] +END_STREAM
           This could force the user agent to serialize subscription requests to the push
           service.
         </t>
+        <t>
+         A user agent that monitors subscriptions for other clients might

```

Comment:
@hruellan - could you provide a clean pull request -and- also respond to @costinm on his [point](https://github.com/webpush-wg/webpush-protocol/pull/96#discussion_r65725759)